incubator/cni-genie/templates/clusterrole.yaml

```diff
@@ -0,0 +1,73 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
For RBAC can you please use the `apiVersion` of `rbac.authorization.k8s.io/v1`. RBAC has been GA for quite a few versions now. This applies to all the objects.